Sources/App/Models/Repository.swift

```diff
@@ -131,6 +134,7 @@ final class Repository: Model, Content {
          defaultBranch: String? = nil,
          firstCommitDate: Date? = nil,
          forks: Int = 0,
+         fundingLinks: [Github.FundingLink] = [],
```

Comment:
Admittedly a bit of a nitpick but I think we should avoid "leaking" GH entities into the db model. I'm not proposing a major change here, just to lift `FundingLink` out of the `Github` namespace into a plain model inside `App/Models` - effectively alongside for instance `Release`, which is a similar entity we're getting from GraphQL.

It'll require a little type shuffle and mapping (just like for release). I'll propose the change in a PR.

(Apart from just "feeling" nicer it'll avoid referencing Github types outside GH specific ingestion and will make it easier to add other services alongside it - if we ever end up doing that 😅)